index.js

```diff
@@ -92,20 +198,56 @@ function format(value, options) {
   var unitSeparator = (options && options.unitSeparator) || '';
   var decimalPlaces = (options && options.decimalPlaces !== undefined) ? options.decimalPlaces : 2;
   var fixedDecimals = Boolean(options && options.fixedDecimals);
-  var unit = 'B';
+  var mode = (options && options.mode) || 'metric';
+  var expectedUnit = (options && options.unit);
 
-  if (mag >= map.tb) {
-    unit = 'TB';
-  } else if (mag >= map.gb) {
-    unit = 'GB';
-  } else if (mag >= map.mb) {
-    unit = 'MB';
-  } else if (mag >= map.kb) {
-    unit = 'kB';
+  // Allow the mode synonym
+  var unitMap;
+  if (mode == 'decimal' || mode == 'metric') {
```

Comment:
this could be hidden that in a `_getUnitMap(mode)` function to make it more readable